tests/integration/test_integration_test_hocons.py

```diff
@@ -46,14 +46,43 @@ class TestIntegrationTestHocons(TestCase):
                 # These can be in any order.
                 # Ideally more basic functionality will come first.
                 # Barring that, try to stick to alphabetical order.
-                "music_nerd_pro/combination_responses_with_history_direct.hocon",
+                "basic/music_nerd_pro/combination_responses_with_history_direct.hocon",
                 # List more hocon files as they become available here.
             ]
         )
     )
-    @pytest.mark.timeout(90)  # 90 seconds for this test
     @pytest.mark.integration
-    def test_hocon(self, test_name: str, test_hocon: str):
+    @pytest.mark.integration_basic
+    def test_hocon_industry(self, test_name: str, test_hocon: str):
+        """
+        Test method for a single parameterized test case specified by a hocon file.
+        Arguments to this method are given by the iteration that happens as a result
+        of the magic of the @parameterized.expand annotation above.
+
+        :param test_name: The name of a single test.
+        :param test_hocon: The hocon file of a single data-driven test case.
+        """
+        # Call the guts of the dynamic test driver.
+        # This will expand the test_hocon file name from the expanded list to
+        # include the file basis implied by the __file__ and path_to_basis above.
+        self.DYNAMIC.one_test_hocon(self, test_name, test_hocon)
+
+    @parameterized.expand(
+        DynamicHoconUnitTests.from_hocon_list(
+            [
+                # These can be in any order.
+                # Ideally more basic functionality will come first.
+                # Barring that, try to stick to alphabetical order.
+                "industry/airline_policy/basic_eco_carryon_baggage.hocon",
+                "industry/airline_policy/basic_eco_carryon_person_item.hocon",
+                "industry/airline_policy/basic_eco_checkin_baggage.hocon",
+                # List more hocon files as they become available here.
+            ]
+        )
```

Comment:
Please add my brand-new trick to the end of this declaration here and wherever lists of test hocon files are declared:
```
, skip_on_empty=True)
```

... this will allow tests to be fine even when all of the entries happen to be commented out.
It will be better copy/paste templating going forward.

See https://github.com/cognizant-ai-lab/neuro-san/blob/main/tests/neuro_san/zzz_hocons/test_unit_test_hocons.py#L50
for clarification.